# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../webmin/setup.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ur webmin-1.180_orig/setup.sh webmin-1.180/setup.sh
--- webmin-1.180_orig/setup.sh	2005-01-24 23:15:16.000000000 +0100
+++ webmin-1.180/setup.sh	2005-05-26 11:53:51.524877296 +0200
@@ -55,14 +55,14 @@
 			echo ""
 			exit 1
 		fi
-	else
+	###else
 		# Make sure dest dir is not in use
-		ls "$wadir" | grep -v rpmsave >/dev/null 2>&1
-		if [ "$?" = "0" -a ! -r "$wadir/setup.sh" ]; then
-			echo "ERROR: Installation directory $wadir contains other files"
-			echo ""
-			exit 1
-		fi
+		###ls "$wadir" | grep -v rpmsave >/dev/null 2>&1
+		###if [ "$?" = "0" -a ! -r "$wadir/setup.sh" ]; then
+		###	echo "ERROR: Installation directory $wadir contains other files"
+		###	echo ""
+		###	exit 1
+		###fi
 	fi
 else
 	echo "Installing Webmin in $wadir ..."
